pytensor/graph/op.py

```diff
@@ -502,7 +502,7 @@ def make_py_thunk(
         self,
         node: Apply,
         storage_map: StorageMapType,
-        compute_map: ComputeMapType,
+        compute_map: ComputeMapType | None,
```

Comment:
This compute_map thing is only ever useful for the LazyLinker in conjuction with IfElse. It's something I want to remove once we switch to IfElse creating two inner functions instead. Having the requirement that every python thunk set the compute_map to True adds quite some overhead. Ops that create inner thunks can now pass None